package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "storyblok-nuxt",
-    "version": "1.4.0",
```

Comment:
We use semantic release for the publishing process to npm so please add a compatible commit message with the prefix "docs: " and revert the version change in the package.json 

https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format

Reply:
Because it is not needed. The version get's generated automatically by semantic-release